scripts/geopmpy/io.py

```diff
@@ -948,7 +948,14 @@ def __init__(self, trace_path, use_agent=True):
         old_governor_headers = {'power_budget': 'POWER_BUDGET'}
         old_headers.update(old_governor_headers)
 
-        column_headers = pandas.read_csv(trace_path, sep='|', comment='#', nrows=0, encoding='utf-8').columns.tolist()
+        skiprows = 0
```

Comment:
Please put a comment in here explaining why we have to do this dance on skipping the rows that start with '#' instead of using the pandas ```comment=``` equals option.  My fear is that we'll come back in here some day, look at this new block and think it's not necessary and reintroduce the bug you're fixing.